features/feedback.feature

```diff
@@ -0,0 +1,9 @@
+@disabled_in_icinga
```

Comment:
I'm probably just missing some context since I've been away, but why is this?

Reply:
Oh yeah, that shouldn't have stayed there.